hugr-core/src/hugr/serialize/test.rs

```diff
@@ -41,44 +41,75 @@ struct SerTestingV1 {
     op_def: Option<SimpleOpDef>,
 }
 
-type TestingModel = SerTestingV1;
+struct NamedSchema {
+    name: &'static str,
+    schema: JSONSchema,
+}
+
+impl NamedSchema {
+    pub fn new(name: &'static str, schema: JSONSchema) -> Self {
+        Self { name, schema }
+    }
+
+    pub fn check(&self, val: &serde_json::Value) {
+        if let Err(errors) = self.schema.validate(val) {
+            // errors don't necessarily implement Debug
+            println!("Schema failed to validate: {}", self.name);
```

Comment:
```suggestion
            eprintln!("Schema failed to validate: {}", self.name);
```

Reply:
and likewise for below

Reply:
this one didn't seem to get committed, only the below

Reply:
Oops, fixed.